src/pybind/mgr/volumes/fs/operations/versions/subvolume_v2.py

```diff
@@ -337,11 +337,24 @@ def trash_incarnation_dir(self):
         except cephfs.Error as e:
             raise VolumeException(-e.args[0], e.args[1])
 
-    def remove(self, retainsnaps=False):
+    def safe_to_remove_subvolume_clone(self, internal_cleanup):
```

Comment:
nit: this can be converted to a static method imo.

Reply:
Done